app/lib/notify-templates.lib.js

```diff
@@ -21,6 +21,22 @@ const notifyTemplates = {
       licenceHolderLetter: '4fe80aed-c5dd-44c3-9044-d0289d635019',
       returnsToLetter: '0e535549-99a2-44a9-84a7-589b12d00879'
     }
+  },
+  'abstraction-alerts': {
+    reduceWarning: '27499bbd-e854-4f13-884e-30e0894526b6',
```

Comment:
How do we work these out. 

There are `restriction_type` and `status` in the licence monitoring station in the db. 

Statuses in my db show - 'reduce', 'resume' and 'warning'

Do we need to look at the last type of alert sent ? 

The assumption would be: 

['chosen-alert-type']['status']

This would mean a 'reduce''reduce' = reduce ?